### ISSUE TITLE
False-positive protobuf-type-error on float fields: should accept int/long as well

### ISSUE THREAD

**Issue by nelfin** [nelfin/pylint-protobuf#41]
Ref #40. Given:

```protobuf
// example.proto
syntax = "proto3";
message Example {
  int32 value = 1;
  float fraction = 2;
}
```

```python
# issue40b.py
from example_pb2 import Example
Example(fraction=123)
```

We warn but shouldn't:

```
$ pylint issue40b.py
************* Module issue40b
issue40b.py:2:0: E5903: Field "Example.fraction" is of type 'float' and value 123 will raise TypeError at runtime (protobuf-type-error)
$ python issue40b.py  # no error
```
